From 2dcbd4388be7473c08712f70a800cde0d67c3681 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 22 Jun 2018 13:40:53 +0800
Subject: [PATCH 4516/5242] MLK-19413-5 arm64: fsl-imx8x-arm2.dtsi: Enable
 pixel combiner

commit  b4316e2d9b91840774f3e1be99eceea2cbd30626 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch enables pixel combiner for the i.MX8x ARM2 platform.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
index 2809541..75c5e4e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8x-arm2.dtsi
@@ -748,6 +748,10 @@
 	status = "okay";
 };
 
+&pixel_combiner {
+	status = "okay";
+};
+
 &prg1 {
 	status = "okay";
 };
-- 
1.7.9.5

